pyupgrade.py

```diff
@@ -1359,6 +1384,8 @@ def _visit_comp(self, node: ast.expr) -> None:
     def _visit_simple(self, node: NameOrAttr) -> None:
         if self._is_six(node, SIX_SIMPLE_ATTRS):
             self.six_simple[_ast_to_offset(node)] = node
+        elif self._is_mock(node, self.MOCK_ALL):
+            self.mock_simple.add(_ast_to_offset(node))
```

Comment:
shouldn't this just replace `import mock` with `from unittest import mock`?

Reply:
Sure